src/libcsg/modules/io/lammpsdumpreader.cc

```diff
@@ -161,7 +158,7 @@ void LAMMPSDumpReader::ReadAtoms(Topology &top, string itemline) {
 
   {
     tools::Tokenizer tok(itemline.substr(12), " ");
-    tok.ToVector(fields);
+    fields = tok.ToVector();
```

Comment:
should this `field` assignment be written in a single line together with the `fields` declaration?